src/components/layout/Icon.jsx

```diff
@@ -3,16 +3,31 @@ import React from 'react'
 
 import { ROOT_PATH } from '../../utils/config'
 
-const Icon = ({ svg, ...imgProps }) => (
-  <img
-    alt={svg}
-    src={`${ROOT_PATH}/icons/${svg}.svg`}
+export const getImageUrl = (svg, png) => {
+  let iconUrl
+  svg ? (iconUrl = `${ROOT_PATH}/icons/${svg}.svg`) : (iconUrl = `${ROOT_PATH}/icons/${png}.png`)
```

Comment:
```suggestion
  return svg ? `${ROOT_PATH}/icons/${svg}.svg` : `${ROOT_PATH}/icons/${png}.png`
```

Par conséquent, la fonction n'est plus trop utile, on peut s'en passer.

Reply:
Oui, c'est vrai que ça enlève plein de lignes mais on avait commencé il me semble à extraire la logique des composants en fonction externe au composant et qu'on peut tester unitairement. Je change le code mais ça serait cool qu'on s'aligne en rétro technique